build_webpack/webpack.terminal.config.js

```diff
@@ -2,11 +2,26 @@ const webpack = require('webpack');
 const base = require('./webpack.base.config');
 const merge = require('webpack-merge');
 const nodeExternals = require('webpack-node-externals');
+const addWithsConfig = require('../scripts/npm/with');
+
+const default_arguments = {
+    BROWSER: 'false',
+    SERVER_PORT: process.env.SERVER_PORT,
+    PORT: process.env.PORT,
```

Comment:
here you have `PORT` and in the command line argument you have `SERVER_PORT`. is there a conflict?

Reply:
I don't know. That's how it looked by default.

Reply:
Ok, I will look into it.